frontend/components/ClubPage/Events.js

```diff
@@ -0,0 +1,53 @@
+import { Icon, Text, StrongText, Card } from '../common'
+import { useState, useEffect } from 'react'
+
+const Events = props => {
+  const data = props.data
+
+  const iconStyles = {
+    float: "left",
+    marginRight: "7px"
+  }
+
+  const bigStyles = {
+    fontSize: "13px",
+    fontWeight: "bold"
+  }
+
+  const smallStyles = {
+    fontSize: "13px"
+  }
```

Comment:
Also would be good to use `styled-components` for these styles. We also tend to define font sizes in terms of `rem` an normally a multiple of `0.2rem`. Maybe something like a font size of `0.8rem` would work in this case?